src/sync/installation.test.ts

```diff
@@ -193,6 +205,7 @@ describe("sync/installation", () => {
 		it("should update cursor and continue sync", async () => {
 			const sendSqsMessage = jest.fn();
 			githubUserTokenNock(DatabaseStateCreator.GITHUB_INSTALLATION_ID);
+			githubUserTokenNock(DatabaseStateCreator.GITHUB_INSTALLATION_ID);
```

Comment:
Why do we need to call this twice? 😕

Reply:
Because I removed the FF that was controlling rate_limit endpoint in scheduler. But actually changed that back because don't need to call this when number of subtasks is 0 (and it also broke a lot of tests, which was annoying)